src/bp/core/services/kvs.ts

```diff
@@ -132,9 +134,15 @@ export class KvsService implements sdk.KvsService {
       })
   }
 
-  set = async (key: string, value, path?: string) => {
+  set = async (key: string, value, path?: string, expiryInMs?: string) => {
```

Comment:
For SDK users, it's not really clear that `expiryInMs` can be any string accepted by the `ms()` function.
Also, the name contains `Ms`, but the string can represent durations in other dimensions ("10 days", "2m"). This can be confusing.

Here are some suggestions:
1) We rename the param to `expiry`, make its type `number`, and explain in the SDK doc that this number represents an amount in milliseconds
2) We rename the param to `expiry` and keep the type to `string`. However, we should add a link in the SDK docs pointing to all the supported formats: https://www.npmjs.com/package/ms

Personaly, I prefer option 1), as the SDK users will not have to know the formats supported by the `ms()` function.

Reply:
I'll pick option 2, because it's the format we use everywhere (botpress config, bot config, module config) so users should be used to that